src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -821,28 +821,6 @@ private string ConstructPluginContent(out string srcConfigFilePath, out string d
             StringBuilder initParameters = new StringBuilder();
 
             bool assemblyAndTypeTokensSet = false;
-            if (sessionType == PSSessionType.Workflow)
```

Comment:
We can also remove this method:
private bool IsWorkflowConfigurationType(System.Management.Automation.PowerShell ps)
on line: 3898

Reply:
I'd want to keep the PR as small as possible for fast review and for avoiding breaking something.
We will remove the method in follow PRs.

Reply:
I don't see the point in doing a partial removal.  Workflows are no longer supported and so we might as well remove everything associated with it.  Register-PSSessionConfiguration affects the local machine and configures endpoints for PSCore6, which no longer supports workflows.  I don't see how removing everything workflow breaks functionality.

Reply:
PSSessionType and IsWorkflowConfigurationType() removed.